src/Features/Core/Portable/MetadataAsSource/AbstractMetadataAsSourceService.cs

```diff
@@ -23,11 +23,6 @@ namespace Microsoft.CodeAnalysis.MetadataAsSource
 {
     internal abstract partial class AbstractMetadataAsSourceService : IMetadataAsSourceService
     {
-        private readonly ICodeGenerationService _codeGenerationService;
-
-        protected AbstractMetadataAsSourceService(ICodeGenerationService codeGenerationService)
```

Comment:
📝 This constructor should be retained as `protected`, but changed to no longer have a parameter.

Reply:
Why?